### PR TITLE
namespace labels/annotations with kritis.grafeas.io

### LINKED ISSUE #63: namespace all the labels/annotations
We should namespace all the labels and annotations with something like:

kritis.grafeas.io/<key>

### PR DESCRIPTION
**Pull request by priyawadhwa**
Should fix #63

### CODE REVIEW

File: pkg/kritis/violation/strategy.go [hunk 1]
```diff
@@ -63,8 +64,8 @@ func (a *AnnotationStrategy) HandleViolation(image string, pod *v1.Pod, violatio
 		}
 		annotationValue += string(v.Reason) + "\n"
 	}
-	labels := map[string]string{constants.InvalidImageSecPolicyLabel: labelValue}
-	annotations := map[string]string{constants.InvalidImageSecPolicyLabel: annotationValue}
+	labels := map[string]string{kritis.GroupName: labelValue}
```
Comment (tejal29) [inline pkg/kritis/violation/strategy.go]: The label name should be "{grpname}.{LabelName}".format(kritis.GroupName, constatants.InvalidImageSecPolicy
Label)

Comment (priyawadhwa) [reply pkg/kritis/violation/strategy.go]: done

File: pkg/kritis/admission/admission.go [hunk 2]
```diff
@@ -132,7 +132,8 @@ func checkBreakglass(pod *v1.Pod) bool {
 	if annotations == nil {
 		return false
 	}
-	_, ok := annotations[constants.BREAKGLASS]
+	breakglass := fmt.Sprintf("%s/%s", kritis.GroupName, constants.BREAKGLASS)
```
Comment (dlorenc) [inline pkg/kritis/admission/admission.go]: Might be nice to make all of these labels be constants themselves, declared somewhere globally.

Comment (priyawadhwa) [reply pkg/kritis/admission/admission.go]: done

**Pull request merged**